### ISSUE TITLE
Escrever README.md como tutorial

### ISSUE THREAD

**Issue by JoaoCarabetta** [thenets/brasilio-package#2]
O README.md poderia ter informações que ajudem na captura e um link para um vídeo explicando.

**Comment by thenets** [comment]
Concordo. É importante. Entretanto preciso terminar essas issues antes:

https://github.com/thenets/brasilio-cli/issues/2
https://github.com/thenets/brasilio-cli/issues/12
https://github.com/thenets/brasilio-cli/issues/11
https://github.com/thenets/brasilio-cli/issues/10

Depois disso, vc pode me ajudar a montar um tutorial?

**Comment by JoaoCarabetta** [comment]
Sem problema, só avisar qdo tiver com as issues terminadas

**Comment by thenets** [comment]
Fechou. Avisarei.
